packages/core/upload/admin/src/components/FilterList/index.jsx

```diff
@@ -30,6 +30,15 @@ const FilterList = ({ appliedFilters, filtersSchema, onRemoveFilter }) => {
     const filterObj = filter[attributeName];
     const operator = Object.keys(filterObj)[0];
     let value = filterObj[operator];
+
+    if (Array.isArray(value)) {
+      value = value.join(', ');
+    } else if (typeof value === 'object') {
+      value = Object.values(value).join(', ');
+    } else {
+      value = decodeURIComponent(value);
+    }
```

Comment:
```suggestion
    value = Array.isArray(value) || typeof value === 'object'
      ? Object.values(value).join(', ')
      : decodeURIComponent(value);
```

Reply:
@simotae14 thankyou for the suggestions, I've updated the code and pushed it again